RocketChat/Rocket.Chat
# [Improve] E2E Room Banner

- [] I have added tests that prove my fix is effective or that my feature works (if applicable)
  - [] I have added necessary documentation (if applicable)
  - [] Any dependent changes have been merged and published in downstream modules

## Proposed changes (including videos or screenshots)


E2E Room Banner for encrypted rooms, helps user to enter his e2e keys or reset the keys.
![image](https://user-images.githubusercontent.com/58601732/156178144-e8f40001-074d-4af5-a14f-c14017970e9a.png)



## Issue(s)


## Steps to test or reproduce

1. Create a E2EE room.
2. Don't enter your E2E keys and banner should be visible for encrypted rooms.
## Further comments
